qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/EncounterFormPage.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class EncounterFormPage extends Page {
+
+	private static final String URL = "/admin/encounters/encounter.form";
+	private static final By PATIENT_FIELD= By.id("patientId_id_selection");
+	private static final By FIRST_AUTOCOMPLETE_RESULT = By.cssSelector("body ul li a span");
+	private static final By ENCOUNTER_DATE_TIME_FIELD= By.cssSelector("#content form div:nth-child(1) table tbody tr:nth-child(3) td input[type=text]");
```

Comment:
Use another element locator that can hardly break, locating page elements through `div `often break the test.